src/frontend/src/lib/components/tokens/Tokens.svelte

```diff
@@ -7,11 +7,17 @@
 	import TokensSignedOut from '$lib/components/tokens/TokensSignedOut.svelte';
 	import Header from '$lib/components/ui/Header.svelte';
 	import { authNotSignedIn, authSignedIn } from '$lib/derived/auth.derived';
+	import { testnetsEnabled } from '$lib/derived/settings.derived';
+	import { i18n } from '$lib/stores/i18n.store';
 </script>
 
 <div class:pointer-events-none={$authNotSignedIn} class:blur-[1.5px]={$authNotSignedIn}>
 	<Header>
-		<NetworksSwitcher disabled={$authNotSignedIn} />
+		{#if $testnetsEnabled}
+			<NetworksSwitcher disabled={$authNotSignedIn} />
+		{:else}
+			<h2 class="text-base">{$i18n.tokens.text.title}</h2>
```

Comment:
Spacingwise, those that h2 occupies as much height as the closed `NetworkSwitcher`?

Reply:
not really, but, they occupy the same as the title in the "Transactions" page, so that we have a smooth transition there.

While the switcher will appear ONLY when the user is in the Settings page, so there will be no transition needed, since the user will never see it

Reply:
Alright, thanks for the feedback.